src/assets/locale/en/collections.json

```
@@ -0,0 +1,244 @@
+    "statusUpdated": "Status updated to {{status}}",
+
+    "urlLabel": "URL",
+    "urlRequired": "Please enter a URL",
+    "urlInvalid": "Please enter a valid URL",
+    "titleLabel": "Title (optional)",
+    "titleHint": "Leave blank to auto-detect from the page",
+    "titlePlaceholder": "Article title",
+    "tagsLabel": "Tags",
+    "tagsPlaceholder": "Add tags...",
+    "notesLabel": "Notes (optional)",
+    "notesPlaceholder": "Add any notes about this article...",
```

Biome (2.1.2)

[error] The key notesPlaceholder was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)